### DISCUSSION TITLE
How would you create new headers name from the concatenation of the previous header and the first row?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#130] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/95233522/172996928-801faa2c-538b-4b05-b564-7171f153d7be.JPG)
How would you create new headers name from the concatenation of the previous header and the first row?
-
Saya membuat judul judul kolom baru, tapi dengan kombinasi dari judul lama dan baris pertama. Bagaimana cara efisiennya?

**Comment by ysitta** [comment, marked as answer]
Dear @Hertzspr,

Bapak bisa melakukan hal tersebut dengan code seperti di bawah ini:
Misal kita menggunakan data iris (data yang sudah ada di R)
```
names(iris) <- paste(names(iris), iris[1, ], sep = "_")
iris_clean <- iris[-1,]
```
referensi: https://stackoverflow.com/questions/53707858/merging-two-rows-into-one-header-using-r

Semoga membantu ya Pak

**Reply by Hertzspr** [reply]
Berhasil, terima kasih.
